Tests/Acceptance/AuthorizationEndpointTest.php

```diff
@@ -5,15 +5,16 @@
 namespace Trikoder\Bundle\OAuth2Bundle\Tests\Acceptance;
 
 use DateTimeImmutable;
+use Nyholm\Psr7\Response;
 use Trikoder\Bundle\OAuth2Bundle\Event\AuthorizationRequestResolveEvent;
 use Trikoder\Bundle\OAuth2Bundle\Manager\AccessTokenManagerInterface;
 use Trikoder\Bundle\OAuth2Bundle\Manager\AuthorizationCodeManagerInterface;
 use Trikoder\Bundle\OAuth2Bundle\Manager\ClientManagerInterface;
 use Trikoder\Bundle\OAuth2Bundle\Manager\RefreshTokenManagerInterface;
 use Trikoder\Bundle\OAuth2Bundle\Manager\ScopeManagerInterface;
 use Trikoder\Bundle\OAuth2Bundle\OAuth2Events;
+//use Zend\Diactoros\Response;
```

Comment:
?

Reply:
Zend/Diactoros got deprecated and replaced by the laminas project, but I certainly should remove this comment

Reply:
removed comment